### DISCUSSION TITLE
What's missing in Oqtane for it to have feature parity with DNN?

### DISCUSSION THREAD

**Discussion by abukres** [oqtane/oqtane.framework#1851] [General]
What's missing in Oqtane for it to have feature parity with DNN?

**Comment by sbwalker** [comment]
Actually there is no goal for Oqtane to have feature parity with DNN. The intent behind Oqtane was to strip DNN down to its most essential features and create a new modular application framework based on modern technology, architecture and patterns. Oqtane 3.0 has achieved that goal and now surpasses DNN in a variety of areas ie. cross-platform, multiple database support, shared and isolated tenancy, etc...
